lib/inspec/resources/windows_firewall.rb

```diff
@@ -0,0 +1,108 @@
+require "inspec/resources/powershell"
```

Comment:
Is this `require` needed? Usually another resource will always be already accessible like `inspec.powershell`

Reply:
You were right, they work without the require :)